command/operator_diagnose.go

```diff
@@ -140,6 +152,61 @@ func (c *OperatorDiagnoseCommand) RunWithParsedFlags() int {
 		return 1
 	}
 
+	// Check Listener Information
+	// TODO: Run Diagnose checks on the actual net.Listeners
+
+	disableClustering := config.HAStorage.DisableClustering
+	infoKeys := make([]string, 0, 10)
```

Comment:
why do we specify a capacity here?

Reply:
Copied from server.go.

Reply:
hmm i think i'm a little confused. what's the reason for having that capacity here too? is there a technical reason to have it?

Reply:
Yeah. I think infoKeys here isn't actually relevant -- I kept it mainly to keep the parity between this and server.go, so I could make this dynamic. But I'd want parity with server.go and preexisting code over necessarily cutting out a few extraneous allocations, which is why I went with this.